database/database.go

```diff
@@ -50,7 +50,7 @@ const (
 	syncMode    = "NORMAL"
 )
 
-const dbVersion = 6
+const dbVersion = 7
```

Comment:
i think we should update migration script

Reply:
We will not perform migration for this release.